binaryurl.py

```diff
@@ -8,7 +8,10 @@
 import argparse
 import json
 import sys
-import urllib2
+try:
```

Comment:
the python binary is python 2 on some machine and python3 on others. This way it could work for both.

Reply:
it seems like it would be a better fix to just ensure we're using python 3 on all machines

Reply:
I agree. I was able to fix to python3 binary on all the machines after a few tweaks.